src/decision/CriteriaWeights.jl

```diff
@@ -1,210 +1,199 @@
-Base.@kwdef struct CriteriaWeights{P,P2,P3,N} <: EcoModel
-    seed_wave_stress::P = Param(
+Base.@kwdef struct CriteriaWeights{U1,U2,U3,DU,DO} <: EcoModel
+    seed_wave_stress::U2 = Factor(
         1.0;
-        ptype="real",
-        bounds=(0.0, 1.0),
-        default_bounds=(0.0, 1.0),
```

Comment:
Will changes to sampling functions using `default_bounds` be done in a future PR?

Reply:
Yes. Please read the description above and for the linked PR.